### PR TITLE
DEV: fix incorrect optional chain usages

### PR DESCRIPTION
**Pull request by mrtnzlml**
This Pull Request meets the following criteria:

* [ ] Tests have been added/adjusted for my new feature
* [ ] New Components are registered in index.js of my project
<br/><br/><br/><url>LiveURL: https://orbit-components-flow-unnecessary-optional-chain.surge.sh</url>

### CODE REVIEW

File: src/Card/CardSection/CardSectionContent/index.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ class CardSectionContent extends React.Component<Props, State> {
     contentHeight: 0,
   };
 
-  node: { current: any | HTMLDivElement } = React.createRef();
```
Comment (mrtnzlml) [inline src/Card/CardSection/CardSectionContent/index.js]: This is wrong everywhere you use `{ current: any | Something }`, see: https://github.com/facebook/flow/blob/master/lib/react.js#L246-L247. I left it everywhere there it uncovered additional bugs. Maybe worth checking it.

**Comment by vepor** [comment]
Thanks @mrtnzlml. Just out of curiosity, is it incorrect usage, or just unnecessary? I believe that we needed to use it in previous versions of `flow`.

**Comment by mrtnzlml** [comment]
@vepor Could be considered unnecessary, yes. Even the lint is called like this. I am more strict at myself so this is _incorrect_ for me... 😊

I think this combo of optional chaining and unnecessary chain lint works like this from the beginning.

**Comment by mrtnzlml** [comment]
How about this? :)

**Pull request merged**